src/store.js

```diff
@@ -52,6 +52,10 @@ export default new Vuex.Store({
     allowSavingElementPosition: state => state.allowSavingElementPosition,
   },
   mutations: {
+    setDefaultFlow(state, { source, flow }) {
+      source.set('default', flow);
+      makeDefinitionPropertyReactive(source, 'default', flow);
+    },
```

Comment:
This doesn't need to be on the store - since it doesn't read or set any state in the store. You can basically inline the `source.set` in Modeler.vue where you commit (i.e. `store.commit('setDefaultFlow', { source, flow });`)

Reply:
Removed from store, moved to Modeler.vue